api/role.go

```diff
@@ -133,6 +142,11 @@ type ModifyRoleData struct {
 	// Mentionable specifies whether the role should be mentionable.
 	Mentionable option.NullableBool `json:"mentionable,omitempty"`
 
+	// Icon is the icon of the role. Requires the guild to have the ROLE_ICONS feature. This value is nullable.
+	Icon *Image `json:"icon,omitempty"`
```

Comment:
How would one unset a role icon? `nil` here would just not update it. Would `&Image{}` work? `**Image` might be needed.

Reply:
Don't you just have to set the content to an empty array? https://github.com/diamondburned/arikawa/blob/6dc189519c4e96b9ed5e77c9afd37c8617ae771f/api/image.go#L102

Reply:
Ah, perhaps documenting it would be a better idea. You should document that `NullImage` can be used, and I can add a `var NullImage` in before merging the PR.

Reply:
Sounds good! Will do that in a bit